cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallTest.scala

```diff
@@ -59,10 +62,11 @@ class CallTest extends DocumentingTest {
       p(
         """Most procedures return a stream of records with a fixed set of result fields, similar to how running a Cypher query returns a stream of records.
           |The `YIELD` sub-clause is used to explicitly select which of the available result fields are returned as newly-bound variables from the procedure call to the user or for further processing by the remaining query.
-          |Thus, in order to be able to use `YIELD`, the names (and types) of the output parameters need be known in advance.
+          |Thus, in order to be able to use `YIELD` for explicit columns, the names (and types) of the output parameters need be known in advance.
           |Each yielded result field may optionally be renamed using aliasing (i.e. `resultFieldName AS newName`).
           |All new variables bound by a procedure call are added to the set of variables already bound in the current scope.
-          |It is an error if a procedure call tries to rebind a previously bound variable (i.e. a procedure call cannot shadow a variable that was previously bound in the current scope).""")
+          |It is an error if a procedure call tries to rebind a previously bound variable (i.e. a procedure call cannot shadow a variable that was previously bound in the current scope).
+          |In a standalone procedure call, `YIELD *` can be used to select all non-deprecated columns. In this case, the name of the output parameters does not need to be known in advance.""")
```

Comment:
This paragraph is getting quite long, not sure if or how it could be separated into smaller paragraphs